doc-content/enterprise-only/employee-rostering/er-individualroster-proc.adoc

```diff
@@ -5,7 +5,9 @@ You can view the assigned shifts for particular employees in an employee-centric
 
 .Procedure
 . To view a table of employees and shifts, click the *Availability Roster* tab.
-. In the top right part of the browser window, the *Week of* field lists the week for which the roster is displayed. To view other weeks, use the *<* and *>* buttons next to the *Week of* field. Alternatively, click on the date field and change the date to view the week that includes this date.
-. If the list is longer than 10 employees, the numbers of currently visible employees are displayed in the top right part of the browser window, for example, `1-15 of 34`. Use the *<* and *>* buttons next to the numbers to display other employees in the list. 
++
+In the top left part of the window, you can see the dates for which the roster is displayed. To view other weeks, you can use the *<* and *>* buttons next to the *Week of* field. Alternatively, you can click on the date field and change the date to view the week that includes this date.
```

Comment:
As B4

Here I would make To view other weeks, use.... it's on steps. The put a note for that step:
Alternatively, click the 
Click, never click on. Search for click on.

Reply:
Leaving steps for post GA as previous, but removing click on.